From 602f9ca83c2bdbf511bcb178fcb4b9fc54da955f Mon Sep 17 00:00:00 2001
From: Changqing Li <changqing.li@windriver.com>
Date: Tue, 27 Nov 2018 15:20:46 +0800
Subject: [PATCH] From e3ae56cf0bb4063c31295f45d04e3c504f4b6cc7 Mon Sep 17
 00:00:00 2001 From: Khem Raj <raj.khem@gmail.com> Date: Mon, 24 Apr 2017
 20:41:25 -0700 Subject: [PATCH] autofs: Upgrade to 5.1.2 release

Use pkg-config first to look for external libnsl which is now
split out from glibc, if it does not exist then see if its provided
by glibc itself.

-Khem

Upstream-Status: Pending

update patch to version 5.1.5
Signed-off-by: Changqing Li <changqing.li@windriver.com>
---
 configure.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.in b/configure.in
index 76ecb40..493b9f1 100644
--- a/configure.in
+++ b/configure.in
@@ -238,7 +238,7 @@ AC_SUBST(LIBCLOCK_GETTIME)
 
 PKG_CHECK_MODULES([NSL],[libnsl],,
 [
-AC_CHECK_LIB(nsl, yp_match, NSL_LIBS="-lnsl")
+PKG_CHECK_MODULES([NSL], [libnsl], [], [AC_CHECK_LIB([nsl],[yp_match],[LIBNSL="-lnsl"],[LIBNSL=""])])
 AC_SUBST(NSL_LIBS)
 NSL_CFLAGS=""
 ])
-- 
2.7.4

